server/events/vcs/bitbucketserver/client.go

```diff
@@ -178,6 +178,30 @@ func (b *Client) PullIsApproved(repo models.Repo, pull models.PullRequest) (bool
 	return false, nil
 }
 
+// PullIsMergeable returns true if the merge request has no conflicts and can be merged.
+func (b *Client) PullIsMergeable(repo models.Repo, pull models.PullRequest) (bool, error) {
```

Comment:
Based on https://community.atlassian.com/t5/Bitbucket-questions/Best-way-to-automatically-detect-when-a-pull-request-resulted-in/qaq-p/402120

I don't think this works for bitbucket server.

Maybe leave it out for now until I have time to test it.

Reply:
I don't know either, however I found this which suggests it might work:

https://community.atlassian.com/t5/Bitbucket-questions/Best-way-to-automatically-detect-when-a-pull-request-resulted-in/qaq-p/402120#M28185

Although it looks like that's referring to their cloud offering, IDK.

Reply:
I changed this to just `return true, nil` for now. I left the code in there (commented out), but I can remove it if you prefer.